SIP/sip-44.md

```diff
@@ -0,0 +1,213 @@
+| xt         | no       | Extension        | string       | QmUFc4dyX7TJn5dPxp8CrcDeedoV18owTBUWApYMuF6Koc                        | A valid IPFS CID                                                                                                          | The CID for extended information. The resulting document does not follow any format restrictions, as it completely use case dependent. Good formats are JSON, but also private information in encrypted formats is possible. |
+
+### Custom In-Object Extensions
+
+The mentioned fields are almost all optional. To not overcomplicate minor custom extensions it is possible to add custom fields to this object, as long as the limit of 1000 bytes is not exceeded. As a recommendation a custom field SHOULD have as first character an `x` and SHOULD be only two characters, e.g. `x1`, or `xa` 
+
+> Keep in mind, that custom fields can only be processed by applications, which support it.
+
+_Examples_
+
+
+```json
+{
+  "vs": 1,
+  "tp": "smc",
+  "nm": "Contract A",
+  "ds": "Bla bla",
+  "av": { "QmbWqxBEKC3P8tqsKc98xmWNzrzDtRLMiMPL8wBuTGsMnR": "image/gif" },
+  "bg": { "QmUFc4dyX7TJn5dPxp8CrcDeedoV18owTBUWApYMuF6Koc": "image/jpeg" },
+  "hp": "https://foobar.com",
+  "sr": "^[0-9a-fA-F]{24}$",
+  "al": "@somealias",
+  "x1": "https://github.com/foobar/contracts/blob/main/sources/contract.c",
+}
+```
+
+
+Not recommended but still conformant objects:
+
+
+_No x as first char used_
+```json
+{
+  "vs": 1,
+  "tp": "hum",
+  "nm": "SignumArt Creator",
+  "ds": "Some nice artworks from wy side",
+  "av": { "QmbWqxBEKC3P8tqsKc98xmWNzrzDtRLMiMPL8wBuTGsMnR": "image/gif" },
+  "bg": { "QmUFc4dyX7TJn5dPxp8CrcDeedoV18owTBUWApYMuF6Koc": "image/jpeg" },
+  "tw": "mytwitter"
+}
+```
+
+_More than two characters_
+
+```json
+{
+  "vs": 1,
+  "tp": "hum",
+  "nm": "SignumArt Creator",
+  "ds": "Some nice artworks from wy side",
+  "av": { "QmbWqxBEKC3P8tqsKc98xmWNzrzDtRLMiMPL8wBuTGsMnR": "image/gif" },
+  "bg": { "QmUFc4dyX7TJn5dPxp8CrcDeedoV18owTBUWApYMuF6Koc": "image/jpeg" },
+  "x-tw": "mytwitter"
+}
+```
+
+
+### Payload Size Limit
+
+The JSON object SHOULD be [minimized](https://codebeautify.org/jsonminifier) to reduce the payload.
+It should be noted that the total size of 1000 (one thousand) bytes MUST NOT be exceeded. It is the responsibility of the user to pay attention to the total size. By the fact that many fields are optional, a margin is created.
+Mind that Unicode characters can occupy more than a single byte. 
+
+#### Compression
+
+> Please review this part and comment here - Personally, I (ohager) do not favor compression.
```

Comment:
Still a point for discussion... I tend to not use compression. But would like to here your opinions